src/app/pages/learning-research/components/research-areas.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {Box} from 'boxible';
+import styled from '@emotion/styled';
+import {Section} from '~/pages/learning-research/components/section';
+import {colors, media} from '~/pages/learning-research/theme';
+import {TabAccordionCombo} from '~/pages/learning-research/components/tab-accordion-combo';
+import RawHTML from '~/components/jsx-helpers/raw-html';
+
+export const ResearchSection = ({data: {researchAreasList, researchAreaHeader, researchAreaDescriptionMobile}}) => {
+    return (
+        <Section backgroundColor={colors.lightGrayBackground}>
+            <h2>{researchAreaHeader}</h2>
+            <RawHTML html={researchAreaDescriptionMobile} className='mobile-only' />
+            <ResearchFocusAreas researchAreaList={researchAreasList} />
+        </Section>
+    );
+};
+
+export const ResearchFocusAreas = ({researchAreaList}) => {
+    return (
+        <TabAccordionCombo collapseAll={true}>
+            {researchAreaList.map((researchArea, i) => {
+                return (
+                    <div label={researchArea.researchAreaTitle} selected className='py-2' key={i}>
```

Comment:
I've don't think a `label` attribute is valid on a div tag?  Maybe something in the website's JS code's reading it though?

Reply:
label is used in the `TabAccordionCombo` as a prop